# This adds a file "/patched!" with the contents "yay" to whatever tree it is applied to.
diff --git a/patched! b/patched!
new file mode 100644
index 0000000..fffffff
--- a/patched!
+++ b/patched!
@@ -1,0 +1,1 @@
+yay
